react-design/my-app/src/IndexButtons.js

```diff
@@ -0,0 +1,9 @@
+export default function IndexButtons({ items }) {
+  return (
+    <div>
+      {items.map((item) => (
+        <button key={item}>{items.indexOf(item)}</button>
+      ))}
+    </div>
+  );
+}
```

Comment:
Don't forget that the currently-selected item's button should be highlighted:

---

![image](https://github.com/AaronHe09/c0523-code-solutions/assets/16143771/e3a2bb0d-b2ad-41f8-aefc-ef8dac626680)

Reply:
are you not suppose to use an event handler function for that?